test/test_cuda.py

```diff
@@ -1085,6 +1085,13 @@ def test_type_conversions_same_gpu(self):
         self.assertEqual(x.type(torch.int).get_device(), 1)
         self.assertEqual(x.to(torch.int).get_device(), 1)
 
+    def test_abs_zero(self):
```

Comment:
Do we even test for `abs`’s behavior on CUDA? I found no `test_abs` in this file.

Reply:
It's done along with other operators at

https://github.com/pytorch/pytorch/blob/3314d60a759fa16d142049bed9be454e69c535db/test/test_cuda.py#L569

Reply:
Ok, thanks for letting me know.